crates/noirc_frontend/src/monomorphization/mod.rs

```diff
@@ -260,6 +260,17 @@ impl<'interner> Monomorphizer<'interner> {
         match self.interner.expression(&expr) {
             HirExpression::Ident(ident) => self.ident(ident, expr),
             HirExpression::Literal(HirLiteral::Str(contents)) => Literal(Str(contents)),
+            HirExpression::Literal(HirLiteral::FmtStr(contents, idents)) => {
+                let fields = vecmap(idents.clone(), |ident| {
+                    let definition_info = self.interner.definition(ident.id);
+                    if let Some(expr_id) = definition_info.kind.get_rhs() {
```

Comment:
can you use match or perhaps `unwrap_or_else` , instead of the `if let Some() else {}` ?

Reply:
switched